packages/react-ui/src/widgets/legend/legend-types/LegendProportion.js

```diff
@@ -91,10 +91,10 @@ function LegendProportion({ legend }) {
                 : max}
             </Typography>
             <Typography variant='overline' color='textSecondary'>
-              {step2}
+              {intl.formatNumber(step2, { maximumFractionDigits: 6 })}
             </Typography>
             <Typography variant='overline' color='textSecondary'>
-              {step1}
+              {intl.formatNumber(step1, { maximumFractionDigits: 6 })}
```

Comment:
Note – I chose 6 digits since that's what the min/max passed into this component were already rounded to, but I'm not sure where/when they are rounded. Open to other suggestions.

Reply:
I'm strict about these, i try to use max _significant_ digits.

It would be great to differentiate between `0.0000001` and `0.0000003` and i'm not sure what this particular API will to with these numbers.

Anyway, so number with so many leading/trailing zeros without units or scientific notation are unusable anyway,

Reply:
Good call! Agreed that max significant digits should be a better choice here, I've added it.